### ISSUE TITLE
Searching with `puppetclass ~ <class>` causes undefined method `allows_taxonomy_filtering?' for ForemanPuppet::Extensions::Host:Module

### ISSUE THREAD

**Issue by laugmanuel** [theforeman/foreman_puppet#315]
When searching for specific puppetclasses, the UI reports:
```sh
external method 'search_by_puppetclass' failed with error: undefined method `allows_taxonomy_filtering?' for ForemanPuppet::Extensions::Host:Module
```

Seems like the issue only affects non-admin users.

**Comment by Manisha15** [comment]
Which foreman_puppet version are you using?

**Comment by laugmanuel** [comment]
@Manisha15 can confirm that #341 fixes this issue.

**Comment by nadjaheitmann** [comment]
Fixed by #341
